types/ckeditor__ckeditor5-utils/src/diff.d.ts

```diff
@@ -18,4 +18,6 @@ export default function diff(
     cmp?: (a: string, b: string) => boolean,
 ): Change[];
 
-export type Change = "equal" | "insert" | "delete";
+type Change = "equal" | "insert" | "delete";
```

Comment:
`Change` is not exported according to docs.

Reply:
`Change` is just a type and has no runtime representation, having it exported means that someone can import and reuse the type without having to rewrite it.